fern/openapi-ir/definition/example.yml

```diff
@@ -0,0 +1,117 @@
+types:
+  SchemaInstanceId:
+    docs: |
+      Uniquely identifies an instance of a schema. For nested instances,
+      the id will be a dot-delimited path, e.g. foo.bar.baz.
+    type: string
+
+  IntExample: integer
+  Int64Example: long
+  FloatExample: double
+  DoubleExample: double
+  StringExample: string
+  DatetimeExample: datetime
+  DateExample: date
+  Base64Example: base64
+  BooleanExample: boolean
+  LiteralExample: string
```

Comment:
not sure how useful the aliases are, could also inline if you prefer. i'm not opinionated here so defer to you

Reply:
Let's in-line em, I agree with the simplicity 👍